usaspending_api/download/tests/integration/test_populate_monthly_delta_files.py

```diff
@@ -0,0 +1,480 @@
+import zipfile
+import datetime
+import pytest
+import os
+
+from django.core.management import call_command
+from os import listdir
+from model_mommy import mommy
+from csv import reader
+
+from usaspending_api.settings import HOST
+from usaspending_api.awards.models import TransactionDelta
+from usaspending_api.common.helpers.generic_helper import generate_test_db_connection_string
+from usaspending_api.download.tests.integration.test_populate_monthly_files import delete_files
```

Comment:
This `delete_files` is annoying when other files exist in the download folder. It also breaks if there are directories.

A much safer process would be to only delete the specific files or to create a new directory and completely remove the directory using something like `recursive_delete()` in `matview_runner.py`

Reply:
Fair enough